contracts/azorius/LinearERC20Voting.sol

```diff
@@ -105,7 +105,7 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent {
      * @param _proposalId id of the Proposal to vote on
      * @param _voteType Proposal support as defined in VoteType (NO, YES, ABSTAIN)
      */
-    function vote(uint256 _proposalId, uint8 _voteType, bytes memory) external {
+    function vote(uint256 _proposalId, uint8 _voteType, bytes memory) external { // TODO what's this unused parameter???
```

Comment:
@tbwebb22 this `bytes memory` is unnamed and seemingly unused, pretty sure we can remove it?